tests/pkg_tinydtls_sock_async/dtls-client.c

```diff
@@ -134,9 +138,19 @@ static void _dtls_handler(sock_dtls_t *sock, sock_async_flags_t type, void *arg)
         _close_sock(sock);
     }
     if (type & SOCK_ASYNC_CONN_RDY) {
-        puts("Session became ready");
+        if (sock_dtls_get_event_session(sock, &session)) {
+            sock_udp_ep_t ep;
+            sock_dtls_session_get_udp_ep(&session, &ep);
+
+            printf("Session became ready: ");
+            print_ipv6_addr_with_port(ep.addr.ipv6, sizeof(ep.addr), ep.port);
```

Comment:
Any reason why you not use `sock_udp_ep_fmt()` from `sys/include/net/sock/util.h` here? Seems like re-inventing the wheel.

Reply:
I just knew the format function when the address is a `ipv6_addr_t` object but didn't see one for an endpoint. Was not my intention to reinvent, just did not found a proper function.